.github/ubiquibot-config.yml

```diff
@@ -31,9 +30,9 @@ priority-labels:
 default-labels:
   - "Time: <1 Hour"
   - "Priority: 0 (Normal)"
-  - "Test"
 payment-permit-max-price: 1000
 comment-incentives: true
 max-concurrent-bounties: 2
 promotion-comment: "\n<h6>If you enjoy the DevPool experience, please follow <a href='https://github.com/ubiquity'>Ubiquity on GitHub</a> and star <a href='https://github.com/ubiquity/devpool-directory'>this repo</a> to show your support. It helps a lot!</h6>"
 register-wallet-with-verification: false
+assistive-pricing: true
```

Comment:
It is unclear to me if it inherits from https://github.com/ubiquity/ubiquibot-config/blob/c6b3938f62b9bcf2701e68fc29b9015f7d56da7f/.github/ubiquibot-config.yml 

Or if we must redundantly specify config properties per repostiory?

Reply:
Repo config properties override org config properties but if a property in repo config is not defined, it will use org config.

Reply:
Yes, as whilefoo stated, if the property is missing in a repo wide config then an org wide property will be used. So there is no need to redundantly specify config properties per repository.

Reply:
We should enable all features on our organization wide config then! I assume this is why `multiplier` isnt working?
- https://github.com/ubiquity/ubiquibot/issues/595#issuecomment-1673185590
- https://github.com/ubiquity/ubiquibot-config/blob/development/.github/ubiquibot-config.yml
- https://github.com/ubiquity/ubiquibot/blob/development/.github/ubiquibot-config.yml